predicates/interval/interval.go

```diff
@@ -0,0 +1,168 @@
+/*
+Package interval implements custom predicates to match routes
+only during some period of time. Package includes three predicates:
```

Comment:
I would cut this paragraph after the first sentence. See how godoc renders package headlines, when listing subdirectories. https://godoc.org/github.com/zalando/skipper